CMakeLists.txt

```diff
@@ -73,22 +73,35 @@ if (DOWNLOAD_DEPENDENCIES)
   # FetchContent downloads and configures dependencies
   message(STATUS "Fetching SDL3 and iniparser. This might take a while...")
   include(FetchContent)
-  if (WINDOWS_STORE)
-    FetchContent_Declare(
-      SDL3
-      GIT_REPOSITORY "https://github.com/Helloyunho/SDL3-uwp.git"
-      GIT_TAG "main"
-      EXCLUDE_FROM_ALL
+
+  if(ANDROID)
+    set(SDL3_SOURCE_DIR "${CMAKE_SOURCE_DIR}/android-project/build/_deps/sdl3-src")
+
+    add_library(SDL3 SHARED IMPORTED)
+    set_target_properties(SDL3 PROPERTIES
+      IMPORTED_LOCATION "${SDL3_SOURCE_DIR}/build-android/${CMAKE_ANDROID_ARCH_ABI}-build/libSDL3.so"
+      INTERFACE_INCLUDE_DIRECTORIES "${SDL3_SOURCE_DIR}/include"
     )
+    add_library(SDL3::SDL3 ALIAS SDL3)
+    add_library(SDL3::Headers ALIAS SDL3)
   else()
```

Comment:
Can't this `if(ANDROID) blabla()` just be:
```cmake
if(ANDROID)
  find_package(SDL3 REQUIRED CONFIG COMPONENTS Shared)
```
Gradle creates, when prefab is enabled, cmake configure files from the json files inside the aar.
You're hard coding relative paths here that can break in the future.